src/hooks/use-suggestions.ts

```diff
@@ -653,6 +653,11 @@ const getOperatorByDataType = (dataType: string) => {
 
 const getOperators = (op: string[]) => op.map((o) => operators[o])
 
+export function getStringOperators() {
+    const keys = getOperatorByDataType('string').filter(key => !key.includes('exist'))
```

Comment:
This will be `EXISTS`, isn't it ?

Reply:
there is also DOESNT-EXIST without S at the end

Reply:
Still, isn't it with upper case ?

Reply:
`operators[key]` is uppercase, `key` isnt